API.md

```diff
@@ -494,6 +544,17 @@ An `HTTP 404 Not Found` status code is returned if the collection exercise event
xerciseId":"e5f41aa9-38e1-4ffd-9f27-d07e27dc21f6","tag":"return_by","timestamp":"2019-08-07T00:00:00.000Z"},{"id":"091a5146-db78-484f-af46-055b29d90df1","collectionExerciseId":"cc9afa9c-6d49-4bdc-8d05-e5b31a55fb36","tag":"mps","timestamp":"2019-09-20T00:00:00.000Z"},{"id":"66ad7848-18b1-4fc0-8ccf-da0b108b4f73","collectionExerciseId":"6ec325ba-6787-442e-8e7c-7b85a231aebb","tag":"return_by","timestamp":"2019-07-17T00:00:00.000Z"},{"id":"aa0a3888-b260-4cc8-8594-7c30805baa80","collectionExerciseId":"29d5eb7e-0b98-4e1c-9566-5f193036992f","tag":"exercise_end","timestamp":"2021-02-28T00:00:00.000Z"},{"id":"bb166eeb-9339-4f91-94a9-da9dd7f27798","collectionExerciseId":"6694f81e-fab8-40cc-a0f5-a99cc40b9a29","tag":"exercise_end","timestamp":"2020-09-30T00:00:00.000Z"},{"id":"3c6c34d4-950e-4ef3-9b12-eecd8795c42c","collectionExerciseId":"032a8b37-6cb4-4fba-adce-afff5bb1b4dc","tag":"ref_period_start","timestamp":"2019-06-01T00:00:00.000Z"},{"id":"195c9379-faf5-499c-ac92-229b2ba9902f","collectionExerciseId":"feaaf1fe-cca9-40a9-bd98-54bc6fd6608b","tag":"mps","timestamp":"2019-08-16T00:00:00.000Z"},{"id":"4e716948-cd02-4b9c-b9bb-7e4c3f5474f7","collectionExerciseId":"8ad14503-49bd-4d30-9c74-604212429788","tag":"reminder","timestamp":"2019-08-13T00:00:00.000Z"},{"id":"40bc7051-1ea6-414a-889e-342d8948223c","collectionExerciseId":"5c3afe3c-5504-4d51-91b5-9ff7fabf7d19","tag":"mps","timestamp":"2019-07-19T00:00:00.000Z"},{"id":"b7d3672d-20d4-4d91-a9e6-f02d473162fd","collectionExerciseId":"51657690-7397-4da1-b48c-5a2f1071b6ae","tag":"exercise_end","timestamp":"2021-04-30T00:00:00.000Z"},{"id":"1861f5d2-a853-45c2-96ad-dacfc459392d","collectionExerciseId":"66e98d19-4a60-4a3d-a4d0-951f0020dd65","tag":"exercise_end","timestamp":"2021-08-31T00:00:00.000Z"},{"id":"e7ead5e6-8be2-4bfe-97c0-d1137608c2c4","collectionExerciseId":"dc08941f-7cb1-4049-bf09-92ab23d10784","tag":"exercise_end","timestamp":"2021-07-31T00:00:00.000Z"},{"id":"d3a61bbf-6753-4c62-ad73-a99be2890544","collectionExerciseId":"0529a56c-d852-4935-8a1e-316db9d928fe","tag":"exercise_end","timestamp":"2021-11-30T00:00:00.000Z"},{"id":"6f1ccffe-39fd-41c4-ae33-5428044b4c9e","collectionExerciseId":"1141b6f1-6f8e-4920-af0d-88a6875eb832","tag":"ref_period_start","timestamp":"2019-12-01T00:00:00.000Z"},{"id":"e5e8ef1d-e500-4a1c-9b88-add8852e95f6","collectionExerciseId":"fcb53bc0-946f-4842-a957-1d6c8fcadcb5","tag":"exercise_end","timestamp":"2020-09-30T00:00:00.000Z"},{"id":"9c361a5e-2a01-4719-a920-259000b5c2e3","collectionExerciseId":"1ad78cb0-b4ff-454b-b807-c03351963df9","tag":"ref_period_end","timestamp":"2019-06-30T00:00:00.000Z"},{"id":"76a76aba-31f8-4184-9e4d-c4c3e6255f05","collectionExerciseId":"d4445b0c-7e61-4b5d-afbc-4eafa6b7d9de","tag":"ref_period_end","timestamp":"2019-11-30T00:00:00.000Z"},{"id":"c9db8f2a-877f-4110-8b3a-5d5a5a1568b4","collectionExerciseId":"631383db-5076-4de9-ae5a-96a30ad6ea5a","tag":"ref_period_end","timestamp":"2019-12-31T00:00:00.000Z"},{"id":"cc79fb97-44af-4e88-9efc-93f70ccf37c0","collection
```

Comment:
This seems way too large to put into documentation - I'd suggest cutting a load of these collexes out